cogip/tools/server_beacon/robot.py

```diff
@@ -76,25 +76,25 @@ async def disconnect():
             logger.info(f"Beacon handler: disconnected from robot {self.robot_id}")
```

Comment:
In `tools: server_beacon: remove display of obstacles` I would add a commit body saying it is for saving CPU.

Reply:
Done